ceph-releases/jewel/ubuntu/14.04/daemon/debug.sh

```diff
@@ -11,6 +11,8 @@ for option in $(comma_to_space ${DEBUG}); do
   case $option in
     verbose)
       echo "VERBOSE: activating bash debugging mode."
+      DEBUG_MS=20
+      echo "VERBOSE: Ceph daemon will now run in debugging level $DEBUG_MS"
```

Comment:
This is unlikely to be the ceph debugging the user needs.  I would suggest instead having a different env var CEPH_ARGS that is passed through, independent of $VERBOSE?